virttest/qemu_devices/qdevices.py

```diff
@@ -656,12 +656,58 @@ def __init__(self, aobject, child_bus=None, is_root=True):
         super(QBlockdevNode, self).__init__(
             "blockdev", {}, aobject, (), child_bus)
 
-        if is_root:
-            self.params['node-name'] = '%s_%s' % ('drive', aobject)
-        else:
-            self.params['node-name'] = '%s_%s' % (self.TYPE, aobject)
+        self._child_nodes = []
+        self._parent_node = None
+        self.is_root = is_root
```

Comment:
we might can remove this line

Reply:
It is class var declare, It will warn instance attribute is_root defined outside __init__ if we remove it.

Reply:
I saw you call self.set_root(is_root) below, which also set selt.is_root = is_root